cudaextender/include/claraparabricks/genomeworks/cudaextender/utils.hpp

```diff
@@ -0,0 +1,150 @@
+/*
+* Copyright 2020 NVIDIA CORPORATION.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#pragma once
+#include <claraparabricks/genomeworks/io/fasta_parser.hpp>
+#include <claraparabricks/genomeworks/cudaextender/extender.hpp>
+
+#include <string>
+#include <vector>
+#include <fstream>
+#include <cassert>
+
+namespace claraparabricks
+{
+
+namespace genomeworks
+{
+
+namespace cudaextender
+{
+
+constexpr int8_t A_NT = 0;
```

Comment:
Is there any drawback in making this an `enum` (except the NUC2)?

Reply:
I did the `enum` change. It leads to a bunch of `static_cast`s. Check it out if it's okay or I can revert.

Reply:
ah, right. sorry. Yes, then revert.

Reply:
Done.